### PR TITLE
Update Fleet/Agent docs for Agent tamper protection (Elastic Defend)

### PR DESCRIPTION
**Pull request by joepeeples**
Contributes to https://github.com/elastic/security-docs/issues/3183 by updating the Fleet & Elastic Agent guide with some notes and links regarding Elastic Defend's Agent tamper protection feature, which is configured and accessed via Agent policy and Fleet UI. Links to [Elastic Security docs](https://www.elastic.co/guide/en/security/master/agent-tamper-protection.html) created via https://github.com/elastic/security-docs/pull/4232.

Previews:
- [Uninstall Elastic Agents from edge hosts](https://ingest-docs_674.docs-preview.app.elstc.co/guide/en/fleet/master/uninstall-elastic-agent.html) — Added note below uninstall Agent commands.
- [Centrally manage Elastic Agents in Fleet](https://ingest-docs_674.docs-preview.app.elstc.co/guide/en/fleet/master/manage-agents-in-fleet.html) — Added "Uninstall tokens" to list of Fleet UI tabs.

### CODE REVIEW

File: docs/en/ingest-management/elastic-agent/uninstall-elastic-agent.asciidoc [hunk 1]
```diff
@@ -21,6 +21,8 @@ If you run into problems, refer to <<fleet-troubleshooting>>.
 If you are using DEB or RPM, you can use the package manager to remove the
 installed package.
 
+NOTE: For hosts enrolled in the {elastic-defend} integration with Agent tamper protection enabled, you'll need to include the uninstall token in the command, using the `--uninstall-token` flag. Refer to the {security-guide}/agent-tamper-protection.html[{elastic-sec} documentation] for more information.
```
Comment (joepeeples) [inline docs/en/ingest-management/elastic-agent/uninstall-elastic-agent.asciidoc:24]:
```suggestion
NOTE: For hosts enrolled in the {elastic-defend} integration with Agent tamper protection enabled, you'll need to include the uninstall token in the command, using the `--uninstall-token` flag. Refer to the {security-guide}/agent-tamper-protection.html[Agent tamper protection docs] for more information.
```

**Review by pierrehilbert — APPROVED** [review]
Thx for this.
cc @amolnater-qasource as you had questions about it.

**Review by kilfoyle — APPROVED** [review]
LGTM! Thanks a lot @joepeeples

**Comment by joepeeples** [comment]
> Joe, do you think it would make sense to also add an entry in this list, just to mention that tamper protection as another benefit?

@kilfoyle I'm thinking no, because tamper protection is only supported for a subset of Agent use cases (only Elastic Defend). If it could be enabled on _all_ Agent policies, it'd probably be worth mentioning, but it doesn't sound like there's any plan for wider support outside of Elastic Defend.

**Comment by kilfoyle** [comment]
> @elastic/amer-fed-education-team kilfoyle I'm thinking no, because tamper protection is only supported for a subset of Agent use cases (only Elastic Defend). If it could be enabled on all Agent policies, it'd probably be worth mentioning, but it doesn't sound like there's any plan for wider support outside of Elastic Defend.

Sounds good! I agree with your reasoning. Thanks Joe!

**Pull request merged**